components/financial_assistance/app/domain/financial_assistance/operations/applications/transformers/application_to/cv3_application.rb

```diff
@@ -134,9 +134,12 @@ def encrypt(value)
 
             def applicant_qnc_code(applicant)
               return true if use_defaults_for_qnc_and_five_year_bar_data?
-              return applicant.qualified_non_citizen if applicant.qualified_non_citizen.present?
 
-              applicant.eligible_immigration_status ? true : false
+              # Check qualified_non_citizen directly, no need for .present? as it's a boolean
```

Comment:
I find this comment a bit confusing because it references `.present?`, which has been removed, and you would need to see the old code to understand that it used to be there.  Also, the code isn't exactly checking that field directly-- its' checking if it's nil.  IMO, we could just remove the comments since the code is very readable on its own.